### PR TITLE
Document pianoroll

### PR DESCRIPTION
**Pull request by ilesinge**
Add pianoroll, color and label function documentation for inline usage (reference tab or autocomplete)

Object key parameters were documented as classic parameters, for ease of formatting

### CODE REVIEW

File: packages/core/pianoroll.mjs [hunk 1]
```diff
@@ -56,6 +56,40 @@ Pattern.prototype.pianoroll = function (options = {}) {
...
+
+/**
+ * Displays a midi-style piano roll
+ *
+ * @name pianoroll
+ * @param {Object} options Object containing all the optional following parameters as key value pairs:
+ * @param {integer} cycles defaults to 4 - number of cycles to be displayed at the same time
+ * @param {number} playhead 0 to 1, defaults to 0.5 - location of the active notes on the time axis
+ * @param {integer} vertical 0 (default) or 1 - displays the roll vertically
+ * @param {boolean} labels false (default) or true - displays labels on individual notes (see the label function)
```
Comment (felixroos) [inline packages/core/pianoroll.mjs:68]: all booleans can either be 0 | 1 or false | true

Comment (ilesinge) [reply packages/core/pianoroll.mjs:68]: I guess we can document them all as boolean : 1/0 ?

**Comment by ilesinge** [comment]
@jarmitage would you like to review?

File: packages/core/pianoroll.mjs (hunk 1 above)
Comment (felixroos) [reply packages/core/pianoroll.mjs]: looks good

**Pull request merged**